### ISSUE TITLE
trimTrailingSlash having issue to import

### ISSUE THREAD

**Issue by maxtrunk** [honojs/hono#2482]
### What version of Hono are you using?

4.2.2

### What runtime/platform is your app running on?

Bun

### What steps can reproduce the bug?

Document: https://hono.dev/middleware/builtin/trailing-slash
Issue: Getting the error when test the code locally.

```
import { trimTrailingSlash } from 'hono/trailing-slash'
app.use(trimTrailingSlash())
```
After run `bun run dev`, the following error shows up:

```
[1] /Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:859
[1]     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
[1]            ^
[1] TSError: ⨯ Unable to compile TypeScript:
[1] index.ts(13,35): error TS2307: Cannot find module 'hono/trailing-slash' or its corresponding type declarations.
[1] 
[1]     at createTSError (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:859:12)
[1]     at reportTSError (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:863:19)
[1]     at getOutput (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1077:36)
[1]     at Object.compile (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1433:41)
[1]     at Module.m._compile (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1617:30)
[1]     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
[1]     at Object.require.extensions.<computed> [as .ts] (/Users/maxtrunk/Project_Hono_Demo/node_modules/ts-node/src/index.ts:1621:12)
[1]     at Module.load (node:internal/modules/cjs/loader:1207:32)
[1]     at Function.Module._load (node:internal/modules/cjs/loader:1023:12)
[1]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) {
[1]   diagnosticCodes: [ 2307 ]
[1] }
```

**Comment by yusukebe** [comment]
Hi @maxtrunk 

This is a bug, and I'll fix it soon. Thanks!

**Comment by yusukebe** [comment]
@maxtrunk 

I've released the new version, `4.2.3,` which includes a fix for this issue. Could you check it?

**Comment by maxtrunk** [comment]
Works! Thank you @yusukebe
